docs/pages/machine-id/introduction.mdx

```diff
@@ -0,0 +1,51 @@
+---
+title: Machine ID
+description: Teleport Machine ID introduction, demo and resources.
+---
+
+# Machine ID
+
+Machine ID is a fully automated Certificate Authority designed to
+programmatically issue and renew short-lived certificates to any service
+accounts, with fine-grained role-based access controls and audit built in.
+
+Some of the things you can do with Machine ID:
+
+* Machines can retrieve short-lived certificates for CI/CD pipelines.
+* Machines hsort lived certs for db in applications.
+* Configure role-based access controls and locking for machines.
+* Capture database access events as well as query activity in the audit log.
```

Comment:
Feels like this line is from somewhere else.

Reply:
Yeah, some other product we have?